extensions/ql-vscode/src/extension.ts

```diff
@@ -623,9 +625,15 @@ async function activateWithInstalledDistribution(
 
   void extLogger.log("Initializing variant analysis manager.");
 
-  const app = new ExtensionApp(ctx);
-  const dbModule = await initializeDbModule(app);
-  ctx.subscriptions.push(dbModule);
+  // We only want to initialize the new db panel when the newQueryRunExperience is enabled
+  let dbModule;
+  if (isCanary() && isNewQueryRunExperienceEnabled()) {
+    const app = new ExtensionApp(ctx);
```

Comment:
(Optional) This line can be moved up a bit (and actually it should move up - we should always create an `app`).

Reply:
Why? We don't need it anywhere else in that file I think

Reply:
It's not currently used anywhere else because it's a new concept. `app` is a container of information that is meant to be used across the codebase. There's is nothing "database" specific about `app`.

Reply:
Ok makes sense!